tests/run/pure_py3.py

```diff
@@ -20,7 +20,7 @@ def test_return_type(n: cython.int) -> cython.double:
     return n if is_compiled else float(n)
 
 
-def test_struct(n: cython.int, x: cython.double) -> MyStruct2:
+def test_struct(n: cython.int, x: cython.double) -> MyStruct2:  # type: ignore
```

Comment:
I think it would also be good to write down the reason for ignoring.

Reply:
Good point, I'll get to that.